Tests/Snapshotting+presentationLayer.swift

```diff
@@ -0,0 +1,41 @@
+// Created by Cal Stephens on 12/15/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import SnapshotTesting
+import UIKit
+import XCTest
+
+extension Snapshotting where Value == UIView, Format == UIImage {
+  /// Captures an image of the input `UIView`'s `layer.presentation()`,
+  /// which reflects the current state of any in-flight animations.
+  static func imageOfPresentationLayer(
+    precision: Float = 1,
+    traits: UITraitCollection = .init())
+    -> Snapshotting<UIView, UIImage>
+  {
+    SimplySnapshotting
+      .image(precision: precision, scale: traits.displayScale)
+      .pullback { (view: UIView) -> UIImage in
+        // Place the view in an on-screen window and then
+        // have Core Animation update the screen synchronously
+        let window = UIWindow()
```

Comment:
Should we give the window a bounds equal to the view.bounds? Seems like a size-zero window could cause some weirdness